examples/meta/src/evaluation/cross_validation_multiple_kernel_learning_weights_storage.sg

```diff
@@ -46,9 +46,10 @@ real stddev = result.get_real("std_dev")
 #![get_results]
 
 #![get_fold_machine]
-CrossValidationStorage obs = mkl_obs.get_observation(0)
-CrossValidationFoldStorage fold = obs.get_fold(0)
-Machine machine_mkl = as_machine(fold.get_trained_machine())
+ObservedValue obs_v = mkl_obs.get_observation(0)
+EvaluationResult cross_validation_run = obs_v.get("cross_validation_run")
```

Comment:
I've seen that Java and C# tests fail because they have no way to implicitly convert `SGObject` to `EvaluationResult`. Should I add a new method `as_evaluation_result()`? @karlnapf @vigsterkr

Reply:
and btw, where are `as_kernel`/`as_machine` defined? :/ because I'm not able to find them inside shogun apparently...

Reply:
nevermind. found them